### PR TITLE
chore: add codeowners file

### PR DESCRIPTION
**Pull request by korvin89**
Add owners for `List` and `Select` components.
[More information about codeowners](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners).

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,2 @@
+/src/components/List @korvin89
+/src/components/unstable/Select @korvin89
```
Comment (ogonkov) [inline CODEOWNERS:2]: Should i suggest other components owners here? or we not going to merge this PR?

Comment (korvin89) [reply CODEOWNERS:2]: Sure, you can add other components into this PR

Comment (ogonkov) [reply CODEOWNERS:2]: Initial author that i can find so far

Card @Lunory 
Skeleton @pseudopirate 
Table @mfyodorov 
Toaster @korvin89 
Tooltip @pseudopirate

Comment (ogonkov) [reply CODEOWNERS:2]: List @ma-efremoff

Comment (mfyodorov) [reply CODEOWNERS:2]: and what about base components (Input, Radio, Checkbox etc)?

**Review by mfyodorov — APPROVED** [review]

**Pull request merged**